src/omero_zarr/masks.py

```diff
@@ -1,22 +1,28 @@
-import omero.clients  # noqa
-from omero.model import MaskI
-from omero.rtypes import unwrap
+import argparse
 from collections import defaultdict
 from fileinput import input
-import numpy as np
-import zarr
+from typing import Dict, List, Set, Tuple
 
+import numpy as np
+import omero.clients  # noqa
+from ome_zarr.data import write_multiscale
```

Comment:
In the same spirit as https://github.com/ome/ome-zarr-py/pull/45#pullrequestreview-482203456, we will likely need to define the minimal version of `ome_zarr` in `setup.py` as this depends on upstream API additions.

Reply:
Agreed. I wasn't planning on pushing a release of ome_zarr without color unless there's a pressing need in order to prevent intermediate versions from being created. (Note: this repository is still in need of tests & GitHub actions to _detect_ that dependency.)